lib/openshift-rest-client.js

```diff
@@ -65,23 +66,24 @@ function openshiftClient (settings = {}) {
   const client = {};
 
   Object.assign(client, bindModule(client, {
-    buildconfigs: buildconfigs,
-    builds: builds,
-    clusterrolebindings: clusterrolebindings,
-    configmaps: configMaps,
-    deploymentconfigs: deploymentconfigs,
-    events: events,
-    groups: groups,
-    imagestreams: imagestreams,
-    persistentvolumeclaims: persistentvolumeclaims,
-    pods: pods,
-    projectrequests: projectrequests,
-    projects: projects,
-    replicationcontrollers: replicationcontrollers,
-    rolebindings: rolebindings,
-    routes: routes,
-    secrets: secrets,
-    services: services
+    buildconfigs,
```

Comment:
heh, i actually have this changed in my branch where i'm switching to XO, so +1 :)